PicIt/models/PicItSettings.swift

```diff
@@ -0,0 +1,18 @@
+//
```

Comment:
This file is created to hold all the settings and constants ... not 100% happy with it as it seems a bit too convoluted.